frontend/src/components/DataCard/SuggestedLabels.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useRef, useCallback } from "react";
```

Comment:
I'm still getting the React error from this component some, although I can't consistently replicate it.